public/app/plugins/panel/xychart/types.ts

```diff
@@ -24,6 +37,26 @@ export interface LegendInfo {
   openEditor?: (evt: any) => void;
 }
 
+export interface ScatterFieldConfig extends GeneratedScatterFieldConfig {
+  pointColor?: ColorDimensionConfig;
+  pointSize?: ScaleDimensionConfig;
+  labelValue?: TextDimensionConfig;
+  lineColor?: ColorDimensionConfig;
```

Comment:
```suggestion
  lineColor?: ColorDimensionConfig;
```
These fields are already present on `GeneratedScatterFieldConfig`, so they shouldn't be necessary here.

Reply:
Ah, nice. I first had the dimension schemas in the same file as the other schema. I guess this isn't necessary any more with it being moved to common. Thanks!

Reply:
This also let me clean up the defaultScatterFieldConfig. Very nice!